rb/lib/selenium/webdriver/common/zipper.rb

```diff
@@ -18,6 +18,7 @@
 # under the License.
 
 require 'zip'
+require 'zip/version' # Not required automatically
```

Comment:
I proposed to do this automatically upstream at https://github.com/rubyzip/rubyzip/pull/637

But anyways, this will have to stay for the forseeable future.